actions.yaml

```diff
@@ -20,5 +20,21 @@ set-tls-private-key:
   required:
     - category
 
-get-admin-secrets:
-  description: Get the admin user password and cert chain used by charm.
+set-password:
+  description: Change the admin user's password, which is used by charm. It is for internal charm users and SHOULD NOT be used by applications.
+  params:
+    username:
+      type: string
+      description: The username, the default value 'admin'. Possible values - admin.
+      default: admin
+    password:
+      type: string
+      description: The password will be auto-generated if this option is not set.
+
+get-password:
+  description: Fetch the system user's password and CA chain, which is used by charm. It is for internal charm users and SHOULD NOT be used by applications.
```

Comment:
Stating that these credentials are only for internal charm users makes me question the purpose of these actions. Maybe it should be made more explicit that these are for internal charm use, and for juju admins to interface directly with the application. If an application wants  admin access, it should set `extra-user-roles="admin"` in a client relation, rather than accessing this.

Reply:
I think this text is related to the spec DA021, and in the spirit of staying consistent with the other charms, we may want to keep it as-is unless / until we update the specs.